scripts/runfabtests.sh

```diff
@@ -393,18 +393,21 @@ function complex_test {
 }
 
 function main {
+	local complex_cfg="quick"
+
 	if [[ $1 == "quick" ]]; then
 		local -r tests="unit simple short"
-		local complex_cfg=$1
 	else
 		local -r tests=$(echo $1 | sed 's/all/unit,simple,standard,complex/g' | tr ',' ' ')
 		if [[ $1 == "all" ]]; then
 			local complex_cfg=$1
-		else
-			local complex_cfg=$COMPLEX_CFG
 		fi
 	fi
 
+	if [[ -n "$COMPLEX_CFG" ]]; then
+		local complex_cfg="$COMPLEX_CFG"
```

Comment:
I don't know Bash's scoping rules: does the `local` in this block declaration make a new `complex_cfg` (different than the one declared on new line 396)?